packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -489,7 +490,7 @@ const VirtualListBaseFactory = (type) => {
 							const {dataSize, focusableScrollbar} = this.props;
 							const column = index % dimensionToExtent;
 							const row = (index - column) % dataSize / dimensionToExtent;
-							const isLeaving = directions.up && row === 0 ||
+							isLeaving = directions.up && row === 0 ||
```

Comment:
It seems to me that VirtualList worked well with `(directions.up || directions.right) && row === 0` condition. Spotlight could be on the 'x' close button when 5-way Right as well as 5-way Up is pressed on the last item.

Reply:
Good point. I adjusted the behavior specified in the lines below (regarding considerations needed when `directions.right` is `true`), since there is more to consider under different scenarios.